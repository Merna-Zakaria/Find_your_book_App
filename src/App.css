.customHeight {
  height: 450px;
  min-height: 450px;
}

a {
  text-decoration: none !important;
  color: gray !important;
}

.logoutBtn {
  color: #0dcaf0 !important;
}
.loginImg {
  width: -webkit-fill-available;
}
.blueBg{
  background-color: #effafc;
}