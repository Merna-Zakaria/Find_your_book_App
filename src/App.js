import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import Routes from "./routes";
import "./App.css";

function App() {
  return (
    <div className="appWrapper">
      <Routes />
    </div>

    //another way: using useRoutes() hooks t handle routes
    // <div className="appWrapper">
    //   <Router>
    //     <Routes />
    //   </Router>
    // </div>
  );
}

export default App;
