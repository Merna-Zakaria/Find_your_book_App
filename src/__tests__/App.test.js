import React from 'react';
import App from '../App';
import Header from '../sharedComponents/header';
import  {create}  from 'react-test-renderer';
import renderer from 'react-test-renderer'
 
// describe('My first snapshot',()=>{
//     test('testing App', () => {
//     let tree = create(<App />)
//     expect(tree.toJSON()).toMatchSnapshot();
//   })
// })
        // it('renders correctly', () => {
        //     const tree = renderer
        //       .create(<App />)
        //       .toJSON();
        //     expect(tree).toMatchSnapshot();
        //   });



        describe('My first snapshot',()=>{
    test('testing App', () => {
    let tree = create(<Header />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})