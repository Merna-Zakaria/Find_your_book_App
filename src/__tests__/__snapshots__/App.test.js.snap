// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot testing App 1`] = `
<div
  className="mb-5"
>
  <nav
    className="navbar navbar-expand navbar-dark bg-primary"
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      Navbar
    </a>
    <div
      className="mr-auto navbar-nav"
      onKeyDown={[Function]}
    >
      <a
        aria-selected={false}
        className="nav-link"
        data-rb-event-key="#home"
        href="#home"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="tab"
        tabIndex={-1}
      >
        Home
      </a>
      <a
        aria-selected={false}
        className="nav-link"
        data-rb-event-key="#features"
        href="#features"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="tab"
        tabIndex={-1}
      >
        Features
      </a>
      <a
        aria-selected={false}
        className="nav-link"
        data-rb-event-key="#pricing"
        href="#pricing"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="tab"
        tabIndex={-1}
      >
        Pricing
      </a>
    </div>
    <button
      className="btn btn-outline-light"
      disabled={false}
      type="button"
    >
      Login
    </button>
  </nav>
</div>
`;
